coding-concepts.js

```diff
@@ -37,23 +37,23 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will log Ruby
+// b) Verify and explain: This happend because the log is looking for the string at the 1st index within the array, as apposed to the 1st character in the string.
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+// console.log(weekendDays[0].toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will uppercase both strings.
+// b) Verify and explain: My understanding was .toUpperCase would uppercase all strings, but in this case it only uppercased "saturday" and I'm not sure why.
```

Comment:
It looks like you added [0] after weekDays which will access the value at the first index of the array.  running weekDays.toUpperCase() would throw an error since .toUpperCase() only works on strings and not on arrays.  To capitalize all values in the array, iteration would be needed to change each string.